test/e2e/features/step-definitions/cancel.js

```diff
@@ -31,6 +31,8 @@ Given('I book a room', async function() {
   const element = await this.waitUntilElement(By.xpath('//h2[contains(text(),"Black Room")]'))
```

Comment:
similar to the test above, think you need to add another this.driver.sleep(2000) before this line - when I run the cucumber tests, it fails without it.